### PR TITLE
updated emissions documentation

### PR DESCRIPTION
**Pull request by rawadelkontar**
### Checklist

- [ ] Documentation has been modified appropriately
- [ ] An ISSUE has been created that this is addressing. Issues will get added to the Change Log when the change_log.rb script is run
- [ ] This branch is up-to-date with develop

### CODE REVIEW

File: workflows/carbon_emissions.md [hunk 1]
```diff
@@ -238,39 +241,81 @@ When adding emissions inputs in the project properties, URBANopt will apply thes
...
   }
 ```
-## Outputs
+# Natural Gas, Propane, and FuelOil #2 Emissions
+
+Emisison factors for natural gas, propane, and fueloil#2 are based on EPA eGRID data and calculated using 20-year GWP horizon based on ASHRAE 189.1 :
+	
+	natural gas :  277.358126 KG/MWH
+	propane : 323.896704 KG/MWH
+	Fueloil #2 : 294.962046 KG/MWH
```
Comment (nllong) [inline workflows/carbon_emissions.md:250]:
```suggestion
	Fuel oil #2 : 294.962046 KG/MWH
```

File: workflows/carbon_emissions.md [hunk 2]
```diff
@@ -238,39 +241,81 @@ When adding emissions inputs in the project properties, URBANopt will apply thes
...
+
+These emission factors are then multiplied by the associated facility total energy use profile for the corresponding fuel type to calculate emissions in metric tons (mt) and emissions intensity in kgCO2e/sqft.
 
-Emissions results are reported in URBANopt reports. For each feature, timeseries results are reported in the CSV report and aggregate values are reported in URBANopt JSON reports. Bellow are descriptions the emissions results variables.
+# Outputs
+
+Emissions results are reported in URBANopt reports. For each feature, timeseries results are reported in the CSV report and aggregate values are reported in URBANopt JSON reports. Bellow are descriptions for the emissions results variables.
```
Comment (nllong) [inline workflows/carbon_emissions.md:256]:
```suggestion
Emissions results are reported in URBANopt reports. For each feature, time series results are reported in the CSV report and aggregate values are reported in URBANopt JSON reports. Below are descriptions for the emissions results variables.
```

File: workflows/carbon_emissions.md [hunk 3]
```diff
@@ -238,39 +241,81 @@ When adding emissions inputs in the project properties, URBANopt will apply thes
...
+	Future electricity emissions intensity in kg/ft2 calculated based on an annual emissions factor value for the selected future year
+
+	"future_hourly_electricity_emissions_intensity_kg_per_ft2": 
+	Future electricity emissions intensity in kg/ft2 calculated based on hourly emissions factor values for the selected future year
+
+	"historical_annual_electricity_emissions_intensity_kg_per_ft2": 
+	Historical electricity emissions intensity in kg/ft2 calculated based on an annual emissions factor value for the selected historical year
+
+	"historical_hourly_electricity_emissions_intensity_kg_per_ft2":
+	Historical electricity emissions intensity in kg.ft2 calculated based on hourly emissions factor values for the selected historical year
```
Comment (nllong) [inline workflows/carbon_emissions.md:284]:
```suggestion
	Historical electricity emissions intensity in kg/ft2 calculated based on hourly emissions factor values for the selected historical year
```

File: workflows/carbon_emissions.md [hunk 4]
```diff
@@ -238,39 +241,81 @@ When adding emissions inputs in the project properties, URBANopt will apply thes
...
+
+	"natural_gas_emissions_mt": 
+	Natural Gas emissions in mt calculated by multipling the facility natural gas energy use (MWH) by the natural gas emission factor (277.358126 KG/MWH) and convertd to metric ton(mt)
```
Comment (nllong) [inline workflows/carbon_emissions.md:287]:
```suggestion
	Natural Gas emissions in metric tonnes (mt) were calculated by multiplying the facility's natural gas energy use (MWh) by the natural gas emission factor (277.4 kg/MWH) and converted to metric tonnes (mt)
```
I think tonnes is the right spelling here? Or maybe not. Another opinion is probably needed.

**Review by nllong — CHANGES_REQUESTED** [review]
few tweaks. I'm sure I missed some orthographic updates, so a thorough review is recommended.

File: workflows/carbon_emissions.md [hunk 5]
```diff
@@ -5,12 +5,15 @@ parent: Workflows
 nav_order: 5
 ---
 
-# Carbon Emissions Reporting
+# **Carbon Emissions Reporting**
 
-The following documents the capabilities added to URBANopt&trade; to calculate carbon emissions associated with building electricity use. Emission calculations for other fuel types will be added soon. Note that the modeling capabilities are currently in Beta mode. This means that testing and development are still in progress, and user feedback is welcome.
-Carbon emissions calculations for electricity use are added to URBANopt to enable users to plan/design for low/net zero carbon buildings, neighborhoods, and urban districts. Calculated emissions values are direct emissions that occur from sources controlled or owned by an organization. Users can calculate emissions and compare the results across multiple UO Scenarios.
+The following documents the capabilities added to URBANopt&trade; to calculate carbon emissions associated with building electricity, natural gas, propane and fuel oil number 1 use.
```
Comment (kflemin) [inline workflows/carbon_emissions.md:10]: Number 2 or number 1?

**Review by kflemin — APPROVED** [review]

File: workflows/carbon_emissions.md [hunk 6]
```diff
@@ -238,39 +241,81 @@ When adding emissions inputs in the project properties, URBANopt will apply thes
     "emissions_annual_historical_year": "2019"
   }
 ```
-## Outputs
+# Natural Gas, Propane, and FuelOil #2 Emissions
+
+Emission factors for natural gas, propane, and fuel oil no. 2 are based on EPA eGRID data and calculated using a 20-year global warming potential (GWP) horizon based on ASHRAE 189.1 :
+	
+	natural gas :  277.358126 KG/MWH
```
Comment (nllong) [inline workflows/carbon_emissions.md:248]:
```suggestion
	natural gas :  277.358126 kg/MWh
```

is this suppose to be kg/MWh? I'm not sure what KG/MWH is. @rawadelkontar

**Pull request merged**